### DISCUSSION TITLE
How do I onboard an additional GitHub repo?

### DISCUSSION THREAD

**Discussion by allanlewis** [renovatebot/renovate#28340] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I have Renovate enabled for a bunch of repos in my GitHub org. I want to enable it for an additional repo but I can't work out how to do it. I added the new repo to the list of selected repositories for the app installation and pushed a valid `renovate.json` about an hour ago but Renovate doesn't list the repo under "Installed Repositories". What else do I need to do to get Renovate to start scanning this new repo?

FWIW I've tested the configuration locally in dry run mode and it generated several updates, as I'd expect.

### Logs (if relevant)

I'm not sure what logs I can get for this issue as the repo doesn't appear on the dashboard 😕

**Comment by rarkins** [comment]
If you picked "Selected repositories" when you installed, then you need to go back to https://github.com/apps/renovate then pick "Configure" next to the account to change the list

**Reply by allanlewis** [reply]
Sorry, I forgot to say I've already done that.

**Reply by allanlewis** [reply]
I've updated the original post with that detail.

**Reply by rarkins** [reply]
In theory, the Mend backend system should have received webhooks when you installed the repo and also when you pushed the file. Please try logging out and back in again, and if that doesn't work then can you tell me the repository name?

**Reply by allanlewis** [reply]
OK - I just logged out of the Renovate portal and then back in: the repo still isn't listed.

The repo is called `Piclo/piclo-infrastructure`.

**Reply by allanlewis** [reply]
The repo still isn't listed by Renovate - I was checking in case it was stuck in a processing queue somewhere.

Should we convert this into an issue?

**Reply by rarkins** [reply, marked as answer]
No, because it's not solvable by code. It was reported to me that it was fixed last night. Perhaps you can try this:
- Remove the repo from selected repositories
- Save
- Add it back to selected repositories
- Save

**Reply by allanlewis** [reply]
Thanks, that worked!
